armada/conf/default.py

```diff
@@ -105,6 +116,21 @@
         help='Defines the format string for \
         %(user_identity)s that is used in logging_context_format_string.'),
 
+    cfg.StrOpt(
+        'project_domain_name',
+        default='default',
+        help='The Keystone project domain name used for authentication.'),
+
+    cfg.StrOpt(
+        'project_name',
+        default='admin',
+        help='The Keystone project name used for authentication.'),
+
+    cfg.PortOpt(
```

Comment:
Like that we are managing the port via the config file can we also do it for the CIDR (ie. 0.0.0.0 )